Assets/MRTK/Core/Providers/BaseCameraSettingsProvider.cs

```diff
@@ -49,6 +49,15 @@ public virtual void ApplyConfiguration()
                 CameraCache.Main.farClipPlane = cameraProfile.FarClipPlaneTransparentDisplay;
                 QualitySettings.SetQualityLevel(cameraProfile.TransparentQualityLevel, false);
             }
+
+            if(Application.isPlaying)
+            {
+                // Move the camera upwards by FloorHeight units if the experience settings explicitly have MRTK initialize the camera to floor height
+                if (!MixedRealityToolkit.Instance.ActiveProfile.ExperienceSettingsProfile.IsNull() && MixedRealityToolkit.Instance.ActiveProfile.ExperienceSettingsProfile.AlignCameraToFloorHeight)
+                {
+                    CameraCache.Main.transform.position = Vector3.up * MixedRealityToolkit.Instance.ActiveProfile.ExperienceSettingsProfile.FloorHeight;
```

Comment:
I'm not sure if this position edit will persist on-device. Unity tends to overwrite any changes, hence why we have to use the MixedRealityPlayspace for teleportation

Reply:
Yeah this behavior is overwritten on both Hololens and Oculus :/

I don't suppose we have some insight on how to redirect the Unity changes?